### PR TITLE
Feature - Issue 560 - Merges tx-manager-lambda into tx-manager

### PR DESCRIPTION
**Pull request by richmahn**
Open for discussion/review. Didn't change the README.rst at all except to add the two sections that are useful from tx-manager-lambda's README.rst and put them at the bottom. README should be worked on once all is properly in place.

Let me know if I missed anything.

### CODE REVIEW

**Comment by richmahn** [comment]
Also to note, this did deploy to the `test` environment with no problem and the API works, e.g. https://test-api.door43.org/tx https://test-api.door43.org/dashboard

**Review by phillip-hopper — CHANGES_REQUESTED** [review]
No changes requested yet. Just wanted to let you know the code review is in progress.

File: deploy.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```
Comment (phillip-hopper) [inline deploy.sh:1]: This file is not executable, mode is 0664.

File: integration_test.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```
Comment (phillip-hopper) [inline integration_test.sh:1]: This file is not executable, mode is 0664.

File: travis-install-apex.sh [hunk 3]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```
Comment (phillip-hopper) [inline travis-install-apex.sh:1]: This file is not executable, mode is 0664.

File: register-modules.sh [hunk 4]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```
Comment (phillip-hopper) [inline register-modules.sh:1]: This file is not executable, mode is 0664.

**Review by phillip-hopper — CHANGES_REQUESTED** [review]
Still waiting for instructions on how to run acceptance tests locally. Currently it fails with this message: 
```
AssertionError: GOGS_USER_TOKEN is missing in environment
```

File: register-modules.sh (hunk 4 above)
Comment (richmahn) [reply register-modules.sh:1]: Ok, fixing file permissions. As for running locally, The tests are all I got, but that is unit testing. I then use the test AWS account and deploy it there when testing everything together.

File: deploy.sh (hunk 1 above)
Comment (richmahn) [reply deploy.sh:1]: fixed

**Review by phillip-hopper — APPROVED** [review]

**Pull request merged**